modules/order/commerce_order.module

```diff
@@ -10,6 +10,17 @@ use Drupal\field\Entity\FieldConfig;
 use Drupal\field\Entity\FieldStorageConfig;
 
 /**
+ * Implements hook_theme().
+ */
+function commerce_order_theme($existing, $type, $theme, $path) {
+  return [
+    'order_edit_form' =>  [
```

Comment:
Should this be commerce_order_edit_form? Are we supposed to namespace the theme functions like that? I know I didn't for the product form, just wondering if that makes sense.